backend/src/main/kotlin/outcobra/server/util/DateOverlabUtil.kt

```diff
@@ -0,0 +1,21 @@
+package outcobra.server.util
```

Comment:
|`line 9`|
|---|
This condition is wrong, this doesn't match when `aEnd < aStart && bEnd < bStart`.